bpf/cpu/cpu.bpf.c

```diff
@@ -406,8 +406,9 @@ static __always_inline void add_stacks(struct bpf_perf_event_data *ctx,
   }
 
   if (method == STACK_WALKING_METHOD_DWARF) {
-    int stack_hash = MurmurHash2((u32 *)unwind_state->stack.addresses,
-                                 MAX_STACK_DEPTH * sizeof(u32), 0);
```

Comment:
Noticed that this was wrong before, the length is 2*the number of elements of the array, as we use an `uint32_t*` to traverse the `uint64_t` array